Assets/Scripts/GameScene/PlayerController.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerController : MonoBehaviour, IPlayerController
+{
+    [SerializeField] private Board board;
+    private PlayerInput _playerInput;
+    private CharacterController _characterController;
+    
+    
+    private Vector2 _movementInput;
+    private Vector3 _playerMovement;
+    private float _playerSpeed = 2.0f;
```

Comment:
this should be configurable from outside of the script, make it `[SerializeField] private...`